pkgs/by-name/bi/bitwarden-directory-connector/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  pkgs,
+  git,
+  python3,
+  pkg-config,
+  libsecret,
+  nodejs_18,
+}: let
+  buildNpmPackage' = buildNpmPackage.override {nodejs = nodejs_18;};
+in
+  buildNpmPackage' rec {
+    pname = "bitwarden-directory-connector";
+    version = "2023.10.0";
+
+    src = fetchFromGitHub {
+      owner = "bitwarden";
+      repo = "directory-connector";
+      rev = "b2bc45137013b258bffda2c2703715cb9f6e687f";
```

Comment:
```suggestion
      rev = "v${version}";
```

Reply:
This is actually what I had originally, before reading that a hash was preferred, I can switch it back no problem

Reply:
I didn't realize a hash was preferred, but I suppose it helps when upstream changes which commit the tag points to. Using a hash does make it slightly harder to review though.

Reply:
Ye, thankfully ye can lock it to a particular hash so it shouldnt be an issue.